### PR TITLE
Guidelines CRUD including UI created per team

### PR DESCRIPTION
**Pull request by egonm12**

### CODE REVIEW

File: app/controllers/guidelines_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+class GuidelinesController < ApplicationController
+  # before_action :set_guideline, only: [:create, :update]
```
Comment (StefanPahlplatz) [inline app/controllers/guidelines_controller.rb]: comment

File: app/views/guidelines/_guidelines_list.html.haml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        %td
+          = g.name
+        %td
+          = g.kudos
+        %td
+          = link_to 'Edit', edit_guideline_path(id: g.id), id: "edit-#{g.id}"
+        %td
+          = link_to 'Remove', guideline_path(id: g.id, team: current_team.slug), method: :delete, id: "rm-#{g.id}",
+            data: {confirm: "This will remove #{g.name}. Are you sure?"}
+  -#.actions
```
Comment (StefanPahlplatz) [inline app/views/guidelines/_guidelines_list.html.haml]: comments

File: app/models/guideline.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+#  name       :string
+#  kudos      :integer
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  team_id    :integer
+#
+
+class Guideline < ActiveRecord::Base
+  validates :name, presence: true, length: { minimum: 4, maximum: 100 }
+  validates :kudos, presence: true
```
Comment (ariejan) [inline app/models/guideline.rb]: Validate numericality, >1, etc?

File: app/models/transaction.rb [hunk 4]
```diff
@@ -126,37 +126,10 @@ def liked_by_user?(user)
...
-     ['Blog tech', 20],
-     ['Call for Proposal for a conference', 20],
-     ['Lunch&Learn', 20],
-     ['Visit conference', 20],
-     ['Be a special help for someone', 10],
-     ['Be a quick help for someone', 5],
-     ['Start a meeting in time', 1]].freeze
-
-  def self.guidelines_between(from, to)
+  def self.guidelines_between(from, to, team_id)
```
Comment (ariejan) [inline app/models/transaction.rb]: I'm not sure what _Guidelines_ for a _Team_ are doing on a _Transaction_. IMHO this should be placed elsewhere.

File: app/views/guidelines/edit.html.haml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+.page
+  .settings-container
+    .title-container
+      %h1.settings-title
+        Guidelines
+      .actions
+        = link_to guidelines_path(team: current_team.slug), method: "get", class: 'btn secondary' do
+          <i class="fa fa-arrow-left" aria-hidden="true"></i> Back to balances
+    %h2.settings-subtitle
+      Edit balance
```
Comment (ariejan) [inline app/views/guidelines/edit.html.haml]: Edit what now?

**Pull request merged**